source/community/release_notes/quince.rst

```diff
@@ -27,14 +31,36 @@ Instructor Experiences
 Administrators & Operators
 **************************
 
+* `feat: feature flag to disable Advanced Settings <https://github.com/openedx/edx-platform/pull/32015>`__
+
+Which MFEs will be included?
+
+* `frontend-app-learner-record: <https://github.com/openedx/frontend-app-learner-record>`__
+* `frontend-app-learner-dashboard: <https://github.com/openedx/frontend-app-learner-dashboard>`__
+
+New Waffle Flagts:
+
+* `feat: add Waffle Flag to disable resetting self-paced deadlines by learners: <https://github.com/openedx/edx-platform/pull/32148>`__
+* `feat: adds waffle flag for show notifications tray: <https://github.com/openedx/edx-platform/pull/32451>`__
+* `feat: add course waffle flag for learner assistant: <https://github.com/openedx/edx-platform/pull/32657>`__
 
 Deprecations & Removals
 ***********************
 
+* `We have deprecated and migrated the openedx/xblock-utils library into openedx/XBlock: <https://github.com/openedx/XBlock/issues/675>`__
+
+* `Most functionality has been removed from the long-deprecated Old Mongo Modulestore. For more details, please: <https://github.com/openedx/public-engineering/issues/62>`__
+
+* `BasicAuthentication as default authentication class in edx-platform: <https://github.com/openedx/edx-platform/issues/33028>`__
+
+* `Remove JWT_AUTH_REFRESH_COOKIE:  <https://github.com/openedx/public-engineering/issues/190>`__
+
+* `feat: allow for forcing asymmetric jwts <https://github.com/openedx/edx-platform/pull/32045>`__
 
 Developer Experience
 ********************
 
+* `feat: Cross Product Recommendations Logic <https://github.com/openedx/edx-platform/pull/32003>`__
```

Comment:
I can't really figure out what this is for. The linked documentation and issues aren't publicly available. I suspect it's related to a 2U-specific feature. I left a question for the author, but if it's not useful for open edX installations, we can leave it out of the release notes. 

We usually reserve Developer Experience for API changes (and fitlers and hooks, which are basically python APIs). I guess this is an API change, but I'm not sure.